rocketmad/models.py

```diff
@@ -975,6 +975,121 @@ def get_nests(swLat, swLng, neLat, neLng, oSwLat=None, oSwLng=None,
+    path_type = db.Column(TINYINT, default=0, nullable=False)
+    name = db.Column(
+        db.String(length=255, collation='utf8mb4_unicode_ci'), nullable=False
+    )
+    description = db.Column(
+        db.String(length=255, collation='utf8mb4_unicode_ci'), nullable=False
+    )
+    version = db.Column(db.Integer, nullable=False)
+    reversible = db.Column(db.BOOLEAN, nullable=False)
+    # submission_time = db.Column(db.DateTime, nullable=False)
+    route_distance_meters = db.Column(db.Integer, nullable=False)
+    route_duration_seconds = db.Column(db.Integer, nullable=False)
+    pins = db.Column(LONGTEXT, nullable=True)
+    tags = db.Column(LONGTEXT, nullable=True)
+    image = db.Column(
+        db.String(length=255, collation='utf8mb4_unicode_ci'), nullable=True
+    )
+    image_border_color_hex = db.Column(
+        db.String(length=8, collation='utf8mb4_unicode_ci'), nullable=True
+    )
+    # route_submission_status = db.Column(TINYINT, default=0, nullable=False)
+    # route_submission_update_time = db.Column(db.DateTime, nullable=False)
+    start_poi_fort_id = db.Column(
+        db.String(length=50, collation='utf8mb4_unicode_ci'), nullable=False
+    )
+    start_poi_latitude = db.Column(DOUBLE(asdecimal=False), nullable=False)
+    start_poi_longitude = db.Column(DOUBLE(asdecimal=False), nullable=False)
+    start_poi_image_url = db.Column(
+        db.String(length=255, collation='utf8mb4_unicode_ci'), nullable=True
+    )
+    end_poi_fort_id = db.Column(
+        db.String(length=50, collation='utf8mb4_unicode_ci'), nullable=False
+    )
+    end_poi_latitude = db.Column(DOUBLE(asdecimal=False), nullable=False)
+    end_poi_longitude = db.Column(DOUBLE(asdecimal=False), nullable=False)
+    end_poi_image_url = db.Column(
+        db.String(length=255, collation='utf8mb4_unicode_ci'), nullable=True
+    )
+    last_updated = db.Column(
+        db.DateTime, default=datetime.utcnow(), nullable=False
+    )
+
+    @staticmethod
+    def get_routes(swLat, swLng, neLat, neLng, oSwLat=None, oSwLng=None,
+                    oNeLat=None, oNeLng=None, timestamp=0, geofences=None,
+                    exclude_geofences=None):
+        # We can filter by the center of a cell,
+        # this deltas can expand the viewport bounds
+        # So cells with center outside the viewport,
+        # but close to it can be rendered
+        # otherwise edges of cells that intersects
+        # with viewport won't be rendered
+        lat_delta = 0.15
+        lng_delta = 0.4
+
+        query = db.session.query(
+            Routes.route_id, Routes.waypoints, Routes.type, Routes.path_type,
+            Routes.name, Routes.description, Routes.reversible,
+            Routes.route_distance_meters, Routes.route_duration_seconds,
+            Routes.start_poi_latitude, Routes.start_poi_longitude,
+            Routes.end_poi_latitude, Routes.end_poi_longitude,
+            Routes.image_border_color_hex, Routes.image, Routes.tags,
```

Comment:
Using this to color the poi-image borders Purple, Blue, Green, Yellow or Red would look nice.

Reply:
I will add the "on click" function to the path.

But, I don't know what you're suggesting for coloring "the poi-image borders".  All I see here is a list of 9 variables.

Reply:
Use the color value from image_border_color_hex to add a border to the images.
Like the blue borders on pokestops and the team color borders around gym images.